app/decorators/user_decorator.rb

```diff
@@ -73,6 +73,25 @@ def link_with_compact_name(options = {})
     link_to compact_name, profile_path(user.screen_name), options
   end
 
+  def gender
+    return if user.gender.blank?
+    return unless Privacy.for(user, 'GENDER').visible
+    return if user.gender == "NOTSPECIFIED"
+    genders = {
+      "MALE"  => "Férfi",
```

Comment:
I think the usage of `'` and `" ` is a bit inconsistent in this file. If there are no interpolation we should use `'`.

Reply:
I agree, please try to use rubocop, its configuration could be improved, but in the most of the cases it's useful.